src/components/SearchBox/SearchBox.js

```diff
@@ -159,7 +159,7 @@ class SearchBox extends Component {
             autoComplete="off"
             autoCorrect="off"
             autoCapitalize="off"
-            spellCheck={false}
+            spellcheck={false}
```

Comment:
I had the same bug in the Preact export of DocSearch and the actual fix is to use:

```suggestion
            spellCheck="false"
```

Reply:
interesting, my fix seems to work too, but I'll try your suggestion